### ISSUE TITLE
Cannot create two agents with different users on the same host

### ISSUE THREAD

**Issue by windsource** [eclipse-ankaios/ankaios#181]
In the tutorial as part of #157 a server and an agent is started using systemd and thus running as root user on a host. Then a second agent is started using a non-root user on the same host. The second agent cannot be started due to the error message:

```shell
❯ ank-agent --name infotainment --server-url http://127.0.0.1:25551             
[2024-02-05T13:52:21Z ERROR] Run folder creation failed. Cannot continue without run folder.: 'Could not create directory "/tmp/ankaios/infotainment_io": permission denied'

❯ ls -la /tmp/ankaios 
total 0
drwxr-xr-x  3 root root  60  5. Feb 10:36 .
drwxrwxrwt 21 root root 640  5. Feb 14:53 ..
drwxr-xr-x  4 root root  80  5. Feb 14:09 agent_A_io
```

The reason for that is that `/tmp/ankaios` is not writable for a non-root user. Maybe that folder should have the sticky bit set as a solution

## Current Behavior

Not possible to start two agents with one started as root first and the other agent started as non-root afterwards on the same host.

## Expected Behavior

It should be possible to start two agents with different users (root and non-root) on the same host.

## Steps to Reproduce

1. Install Ankaios
2. Start server with `sudo systemctl ank-server`
3. Start agent with `sudo systemctl ank-agent`
4. Start a second agent with `ank-agent --name infotainment --server-url http://127.0.0.1:25551`

## Context (Environment)

ank 0.3.0-pre

## Logs

See description

## Final result

**To be filled by the one closing the issue.**

**Comment by windsource** [comment]
Same problem in version 0.2.0.

**Comment by krucod3** [comment]
Let's try to fix this in v0.3. If it gets too big, let's discuss again.

**Comment by lingnoi** [comment]
Issue fixed with #206
